src/drivers/nuopc/cime_config/buildnml

```diff
@@ -337,11 +338,14 @@ def _create_runseq(case, coupling_times):
             # for Q (aquaplanet) compsets
             runseq_input = os.path.join(input_dir, 'nuopc_runseq_Q')
 
-        elif ( (comp_atm == 'datm' and comp_ocn == "mom"  and comp_ice == "dice") or 
-               (comp_atm == 'datm' and comp_ocn == "mom"  and comp_ice == "cice") or
+        elif ( (comp_atm == 'datm' and (comp_ocn == "mom" or comp_ocn == 'pop')  and comp_ice == "dice") or 
```

Comment:
We need to put this in xml and remove it from python source.

Reply:
I agree totally. not sure when. First we need to get the performance up.